modules/postgres/postgres.go

```diff
@@ -182,6 +189,62 @@ func WithSnapshotName(name string) SnapshotOption {
 	}
 }
 
+// WithSSLSettings configures the Postgres server to run with the provided CA Chain
+// This will not function if the corresponding postgres conf is not correctly configured.
+// Namely the paths below must match what is set in the conf file
+func WithSSLSettings(sslSettings SSLSettings) testcontainers.CustomizeRequestOption {
```

Comment:
suggestion: pass in values not struct to avoid ambiguity.
question: should this be renamed as its not ssl settings to me?
```suggestion
func WithSSLCert(caCertFile, certFile, keyFile) testcontainers.CustomizeRequestOption {
```